webapp/components/Registration/RegistrationSlider.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <section class="login-form">
+    <base-card>
+      <template #imageColumn>
+        <a :href="links.ORGANIZATION" :title="$t('login.moreInfo', metadata)" target="_blank">
+          <img class="image" alt="Welcome" src="/img/custom/welcome.svg" />
+        </a>
+      </template>
+
+      <component-slider :sliderData="sliderData">
+        <template #header>
+          <ds-heading size="h2">
+            {{ $t('components.registration.signup.title', metadata) }}
+          </ds-heading>
+        </template>
+
+        <template v-if="['invite-code'].includes(registrationType)" #enter-invite>
```

Comment:
shouldn't it be `registrationType.includes('invite-code')` ?

Reply:
I’ve used the same style as for the other slider components to have a better recognition that’s about the same thing. Could be for sure as well:
```suggestion
        <template v-if="registrationType === 'invite-code'" #enter-invite>
```
 As far I've understand the array comes before the dot selector and `registrationType` is not an array …